### DISCUSSION TITLE
How to open apps "NOT" where the cursor is but where it is called

### DISCUSSION THREAD

**Discussion by mobsenpai** [hyprwm/Hyprland#5638] [Q&A] [answered]
Is there a built-in option in Hyprland to launch applications on the workspace where they were called, rather than where the cursor is currently located? Specifically, if I call an application from a different workspace than the one I am currently on, I would like the application to open on the original workspace. I am aware of Hyprland's intended behavior to open applications on the workspace where the cursor is, but I am seeking an answer to whether there is a toggleable option within Hyprland to achieve this. I am not interested in any hacky solutions if there is no built-in option.

**Comment by vaxerski** [comment, marked as answer]
tldr nope

**Comment by mobsenpai** [comment]
Okay
